rspec-support.gemspec

```diff
@@ -19,7 +19,12 @@ Gem::Specification.new do |spec|
   spec.rdoc_options  = ["--charset=UTF-8"]
   spec.require_paths = ["lib"]
 
-  private_key = File.expand_path('~/.gem/rspec-gem-private_key.pem')
+
+  private_key = ""
+  begin
+    private_key = File.expand_path('~/.gem/rspec-gem-private_key.pem')
+  rescue ArgumentError => e
+  end
```

Comment:
It would be better to wrap all of the signing logic in this block, rather than run a useless (and potentially wrong) if check,  also maybe it's worth checking for the presence of HOME instead rather than rescuing a misleading exception, WDYT